a10_octavia/controller/worker/tasks/a10_network_tasks.py

```diff
@@ -843,13 +843,18 @@ class DeleteVRIDPort(BaseNetworkTask):
     def execute(self, vthunder, vrid_list, subnet, lb_count, member_count):
         vrid = None
         vrid_floating_ip_list = []
+        if not lb_count:
```

Comment:
I think ".count()" function always return integer value.

Reply:
this was experimental while checking for other cases., removing it.